modules/combined/test/tests/contact/4ElemTensionRelease.i

```diff
@@ -64,26 +64,34 @@
 
 [Materials]
   [./stiffStuff1]
-    type = Elastic
-    block = 1
-
-    disp_x = disp_x
-    disp_y = disp_y
-
-    youngs_modulus = 1e6
+    type = ComputeIsotropicElasticityTensor
+    block = '1'
+    youngs_modulus = 1.0e6
     poissons_ratio = 0.3
   [../]
+  [./stiffStuff1_strain]
+    type= ComputeSmallStrain
+    block = '1'
+  [../]
+  [./stiffStuff1_stress]
+    type = ComputeLinearElasticStress
+    block = '1'
+  [../]
 
   [./stiffStuff2]
-    type = Elastic
-    block = 2
-
-    disp_x = disp_x
-    disp_y = disp_y
-
-    youngs_modulus = 1e6
+    type = ComputeIsotropicElasticityTensor
+    block = '2'
+    youngs_modulus = 1.0e6
     poissons_ratio = 0.3
   [../]
+  [./stiffStuff2_strain]
+    type= ComputeSmallStrain
+    block = '2'
+  [../]
+  [./stiffStuff2__stress]
+    type = ComputeLinearElasticStress
+    block = '2'
```

Comment:
as above no need to have two identical materials block restricted to different blocks. There should only be one ComputeSmallStrain and one ComputeLinearElasticStress. The ComputeSmallStrain should be added through the master action.